multi_import/importer.py

```diff
@@ -104,7 +104,7 @@ def normalize_row_data(self, row_data):
                 value = int(value)
 
             if not isinstance(value, string_types):
```

Comment:
I am not sure, if this `isinstance` check is required? Because we are anyway doing something if the value is a `float`, in case of the check above. Seems redundant to check if value is `not a string type`.

Reply:
I'm tempted to leave this as is if that's ok with you. In some cases the data will be a string so it would be redundant to turn a string into a string if we remove it.

Reply:
Yeah cool.